consensus/polybft/polybft.go

```diff
@@ -295,6 +295,9 @@ func (p *Polybft) initRuntime() error {
 		txPool:                p.txPool,
 		bridgeTopic:           p.bridgeTopic,
 		numBlockConfirmations: p.config.NumBlockConfirmations,
+		jsonrpcAddr:           p.config.Config.JSONRPCAddr,
```

Comment:
Why this was not present before? are you sure this can be done safely here without bridge config?

Reply:
Where is this `JSONRPCAddr` config value coming from?

Reply:
It uses the localhost if it is empty.